coderd/database/migrations/000226_provisioner_keys.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE provisioner_keys (
+    id uuid PRIMARY KEY,
+    created_at timestamptz NOT NULL,
+    organization_id uuid NOT NULL REFERENCES organizations (id) ON DELETE CASCADE,
+    name varchar(64) NOT NULL,
+    hashed_secret bytea NOT NULL
+);
+
+CREATE UNIQUE INDEX provisioner_keys_organization_id_name_idx ON provisioner_keys (organization_id, name);
```

Comment:
To make the index case insensitive:
```
CREATE UNIQUE INDEX provisioner_keys_organization_id_name_idx ON provisioner_keys USING btree (organization_id, lower(name));
```

Reply:
I see that you guard the insert with a `lower(name)` on the insert. That is fine, but in workspaces and templates we guard with the `lower(name)` on each query instead. 

For consistency with the others, I think we should do the `lower()` on each query. I will leave it up to you though.